src/vaadin-time-picker.html

```diff
@@ -567,6 +569,16 @@
           this.__updateValue(parsedObj);
         }
 
+        __minChanged(value) {
+          this.min = value;
```

Comment:
this line is not needed, as `value` passed as an argument is already what `this.min` contains

Reply:
Also, are these separate observer methods needed for some reason? What I meant with my previous comment was that you can add `min` and `max` to the [existing multi-property observer](https://github.com/vaadin/vaadin-time-picker/blob/master/src/vaadin-time-picker.html#L326) to have it invoke every time one of these properties change.

Reply:
Oh didn't know about that :man_facepalming: